kalliope/stt/Utils.py

```diff
@@ -81,4 +57,4 @@ def set_callback(self, callback):
         :param callback: callback method
         :return:
         """
-        self.callback = callback
+        self.callback = callback
```

Comment:
we usually comment people to leave a new line. And you, you remove then ;)

Reply:
Ups^^